### PR TITLE
Update troubleshoot.md

### PR DESCRIPTION
**Pull request by RedBrogdon**
Adds a line with an additional troubleshooting suggestion about enabling cookies for sites that embed DartPad.

### CODE REVIEW

File: src/tools/dartpad/troubleshoot.md [hunk 1]
```diff
@@ -25,7 +25,11 @@ then try the following:
 * Check whether you've disabled third-party
   tracking cookies ([Chrome instructions][chrome-cookies]).
   If you've disabled cookies, change your settings to
-  **allow cookies for dartpad.dev.**
+  **allow cookies for dartpad.dev**. If you're using DartPad via embeds
+  on another site, such as the [Dart cheatsheet](https://dart.dev/codelabs/dart-cheatsheet)
+  or the [Implicit Animations codelab](https://flutter.dev/docs/codelabs/implicit-animations),
+  you may need to allow cookies for those sites' domains as well (in this
+  case, **dart.dev** and **flutter.dev**, respectively).
```
Comment (kwalrath) [inline src/tools/dartpad/troubleshoot.md:32]:
```suggestion
  **allow cookies for dartpad.dev**.
  If you're using embedded DartPads,
  such as in the [Dart cheatsheet codelab](/codelabs/dart-cheatsheet) or the
  [implicit animations codelab]({{site.flutter}}/docs/codelabs/implicit-animations),
  you might need to allow cookies for the embedding site domain, as well
  (in these cases, **dart.dev** and **flutter.dev**, respectively).
```

**Review by kwalrath — APPROVED** [review]
LGTM, but I'd like a few mostly invisible changes.

File: src/tools/dartpad/troubleshoot.md (hunk 1 above)
Comment (kwalrath) [reply src/tools/dartpad/troubleshoot.md:32]: I'm going to make this change, since I'm pretty sure @RedBrogdon won't mind.

**Comment by RedBrogdon** [comment]
FWIW, you should always feel free to tweak any text I send you. 😄

**Pull request merged**